### PR TITLE
Support loading cookies from local machine

### PR DESCRIPTION
**Pull request by frostming**
- Introduce `browsercookie` to read cookies from Chrome or Firefox
- Rename the `--is-cn` option as `--cn` to accept more possible values

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 requests
+browsercookie @ git+https://github.com/richardpenman/browsercookie@master
```
Comment (frostming) [inline requirements.txt:2]: The master branch has the newest change that makes it work on Chrome >80

**Comment by yihong0618** [comment]
LGTM
do we need to merge #7 first?

**Comment by yihong0618** [comment]
Can this `PR` merge?

**Comment by frostming** [comment]
> Can this `PR` merge?

Yes, I've rebased the branch and modified the README.

**Review by yihong0618 — APPROVED** [review]

**Pull request merged**